Makefile

```diff
@@ -59,7 +67,15 @@ $(eval $(foreach step,$(STEPS),$(call make_target,$(step),$(MAKE) PAL=$(TARGET))
 
 shell: ENABLE_TTY=-t
 shell:
+ifeq ($(DISABLE_DOCKER),y)
```

Comment:
This is nice to have but not strictly necessary. I will rather not have it to reduce number of if/else checks in the makefile.
The recipe will simply fail as you probably don't have docker.

Reply:
I thought it would be a good idea for newcomers, students or beginners to have an error message in case they're trying to do something that doesn't make sense. 

Do you want me to delete this ?